feat/PRO-2416/upgrade-prime-modular-accounts (#155)
etherspot/transaction-kit: CHANGELOG.md

# Changelog

## [0.13.0] - 2024-06-17

### Added Changes
- `getSDK` include a param to choose to instantiate the Prime SDK instead of the Modular SDK
- Added Etherspot Modular SDK `installModule` and `uninstallModule` to hook `useEtherspotModules`
- Added `isModular` to context `EtherspotContextProvider`

### Breaking Changes

> - Etherspot Modular SDK implemented to Transaction Kit as the default `accountTemplate`

Reviewer: ```suggestion
- Etherspot Modular SDK implemented to TransactionKit as the default `accountTemplate`
```

> - Etherspot Modular SDK implemented to Transaction Kit as the default SDK

Reviewer: ```suggestion
- Etherspot Modular SDK implemented to Transaction Kit as the default `accountTemplate`
```

Reviewer: Note the above will make sense after reading the comments below

Author: Absolutely!

> - The account template in the `EtherspotTransactionKit` component, can only be 'etherspot' if `moddular` is true.

Reviewer: ```suggestion
```

> - Changed the `etherspot-prime` wallet type to `etherpot` wallet type

Reviewer: ```suggestion
- Changed the `etherspot-prime` wallet type to `etherspot` wallet type
```
